packages/weston.rb

```diff
@@ -43,5 +34,9 @@ def self.build
   def self.install
     system "make DESTDIR=#{CREW_DEST_DIR} install"
   end
-end
 
+  def self.postinstall
+    system "libtool --finish #{CREW_LIB_PREFIX}/weston"
+    system "chmod 700 $XDG_RUNTIME_DIR -v"
```

Comment:
This could be considered insecure... Possibly just silence the warning? Also, user `1000` is chronos, even if the warning says it's not.